flask_rebar/swagger_generation/marshmallow_to_swagger.py

```diff
@@ -27,6 +27,13 @@
 from flask_rebar.validation import CommaSeparatedList
 from flask_rebar.swagger_generation import swagger_words as sw
 
+try:
+    EnumSymbol = m.fields.EnumSymbol
+    EnumValue = m.fields.EnumValue
```

Comment:
Why are these fields not documented by marshmallow?

Reply:
Oh, i was right initially, they merged it after the fact https://github.com/marshmallow-code/marshmallow/commit/fa0ee03e82d2b73dc00bdeb75b266aba99be8d0b

Reply:
Can you explain this a bit?
It doesn't seem to me like it will address the originally raise issue that the `Enum` field introduced in marshmallow 3.18 is not supported.